packages/dotcom-server-handlebars/src/helpers/renderReactComponent.ts

```diff
@@ -0,0 +1,38 @@
+import path from 'path'
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+
+export default function renderReactComponent({ hash }) {
+  let modulePath
+
+  if (hash.hasOwnProperty('package')) {
+    modulePath = path.resolve(`node_modules/${hash.package}`)
+  }
+
+  // localRoute is relative to root of app consuming dotcom-server-handlebars.
+  if (hash.hasOwnProperty('localRoute')) {
+    modulePath = path.resolve(hash.localRoute)
```

Comment:
Rename `localRoute` -> `localPath`.

Reply:
Fixed in [this commit](https://github.com/Financial-Times/dotcom-page-kit/pull/746/commits/c55273cfa0c239efd490dcabeeef6815dc299d21).